client-sdk/go/helpers/address.go

```diff
@@ -50,8 +56,17 @@ func ResolveAddress(net *config.Network, address string) (*types.Address, *ethCo
 			// Pool.
 			switch data {
 			case poolRewards:
-				// Reward pool address.
 				return &rewards.RewardPoolAddress, nil, nil
+			case poolCommon:
+				// TODO: Consensus common pool. Should check, if ParaTime is used or not.
```

Comment:
Not sure if I understand this comment :thinking: 
Who will address this TODO?

Reply:
https://github.com/oasisprotocol/oasis-sdk/issues/1508

I'll rephrase.